File: ANGEN/SampleFramework.Controls/CommandManager/Samples/Notification Category/DotMatrix/CS/DotMatrix/MainForm.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Globalization;
using System.Text;
using System.Windows.Forms;

namespace DotMatrix
{
	public partial class MainForm : Form
	{
		private string GetDateMatrixText()
		{
			return DateTime.Now.ToShortDateString();
		}

		private string GetTimeMatrixText()
		{
			return DateTime.Now.ToLongTimeString();
		}

		private void _timer_Tick(object sender, EventArgs e)
		{
			_timeMatrix.Text = this.GetTimeMatrixText();
			_dateMatrix.Text = this.GetDateMatrixText();
		}

		private void _timer2_Tick(object sender, EventArgs e)
		{
			_counterMatrix.Text = (_counter++).ToString("0000", CultureInfo.CurrentUICulture);
			
			if (_counter == 10000)
			{
				_counter = 0;
			}
		}

		private int _counter;

		public MainForm()
		{
			this.InitializeComponent();
			_timeMatrix.Text = this.GetTimeMatrixText();
			_dateMatrix.Text = this.GetDateMatrixText();
			_timer.Enabled = true;
		}
	}
}

File: ANGEN/SampleFramework.Controls/NavigationBar/Samples/Notification Category/DotMatrix/VB/DotMatrix/MainForm.vb
Imports System.Globalization

Public Class MainForm
    Private Sub _timer_Tick(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles _timer.Tick
        _timeMatrix.Text = Me.GetTimeMatrixText()
        _dateMatrix.Text = Me.GetDateMatrixText()
    End Sub

    Private Sub _timer2_Tick(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles _timer2.Tick
        _counter += 1
        _counterMatrix.Text = _counter.ToString("0000", CultureInfo.CurrentUICulture)

        If _counter = 10000 Then
            _counter = 0
        End If
    End Sub

    Private Function GetDateMatrixText() As String
        Return DateTime.Now.ToShortDateString()
    End Function

    Private Function GetTimeMatrixText() As String
        Return DateTime.Now.ToLongTimeString()
    End Function

    Dim _counter As Int32

    Sub New()
        Me.InitializeComponent()
        _timeMatrix.Text = Me.GetTimeMatrixText()
        _dateMatrix.Text = Me.GetDateMatrixText()
        _timer.Enabled = True
    End Sub
End Class
